boa/interpret.py

```diff
@@ -37,6 +38,15 @@ def set_cache_dir(cache_dir="~/.cache/titanoboa"):
     _disk_cache = DiskCache(cache_dir, compiler_version)
 
 
+def create_deployer(*args, **kwargs):
```

Comment:
yea hm, this is going to blow up as soon as the user uses both zksync env and native env in the same program

Reply:
Well people can always create deployers "manually"
Also it should be fine to load contracts and switch deployers in between

Reply:
This method specifically isn't needed. I removed it and added more comments to the set_deployer_class